pkg/ragengine/controllers/ragengine_controller.go

```diff
@@ -39,6 +46,13 @@ import (
 	"sigs.k8s.io/karpenter/pkg/apis/v1beta1"
 )
 
+const (
+	nodePluginInstallTimeout = 60 * time.Second
```

Comment:
Why do we need this const in this PR?

Reply:
delete both for rag engine and workspace